app/javascript/article-form/articleForm.jsx

```diff
@@ -312,6 +312,7 @@ export class ArticleForm extends Component {
         onSubmit={this.onSubmit}
         onInput={this.toggleEdit}
         data-testid="article-form"
```

Comment:
I'm leaving the `data-testid` attribute temporarily as it's used in the `previewArticle.spec.js` file. It's not related work to the bug fix, so I will create a separate PR for it.

Reply:
Removed in #13376.